src/android/java/com/pspdfkit/cordova/PSPDFKitCordovaPlugin.java

```diff
@@ -52,6 +52,9 @@
     private static final int ARG_DOCUMENT_URI = 0;
     private static final int ARG_OPTIONS = 1;
 
+
+    private String secret;
```

Comment:
Please convert this to a local variable inside the `exec()` method. No need to carry passwords as state in the plugin class.